lib/puppet/provider/certs_bootstrap_rpm/katello_ssl_tool.rb

```diff
@@ -43,7 +43,8 @@ def last_rpm
       {'release' => release, 'rpm' => rpm}
     end
 
-    rpms.sort { |a,b| a['release'].to_i <=> b['release'].to_i }.last['rpm']
+    rpm = rpms.sort { |a,b| a['release'].to_i <=> b['release'].to_i }.last
+    rpm['rpm'] if rpm
```

Comment:
based on what it used to be:   https://github.com/Katello/puppet-certs/pull/83/files  it seems like it handles nil okay, but i can skip up above if its nill to be more explicit.